packages/@okta/vuepress-site/docs/concepts/multi-tenancy/index.md

```diff
@@ -0,0 +1,453 @@
+applications.
+
+#### Connecting the spokes to the hubs
+
+Using the Org2Org connector, spokes can add users and give access to shared
+applications and services through the hub.
+
+#### Diagram
+
+A diagram illustrating the hub and spoke configuration is shown below:
+
+<div class="image-at-90">
+
+![Multi-org layout](/img/multi-tenancy/multi-org-configuration-layout.png
+ "Diagram showing how the layout of the hub and spoke model")
+
+</div>
+
+#### More information
+
+To discover more about the hub and spoke architecture visit the following links:
+
+* [Different Ways To Architect Multitenancy in Okta](https://www.okta-from-the-field.com/post/different-ways-to-architect-multitenancy-in-okta)
+* [Okta for Global, Distributed Organizations](https://www.okta.com/resources/whitepaper/okta-for-global-distributed-organizations/)
+* [An Identity Framework for Higher Education Systems](https://www.okta.com/resources/whitepaper/an-identity-framework-for-higher-education-systems/)
+
+## Configuration 3
+
+Hybrid - host tenants in both single and separate orgs
+
+### Summary
+
+Okta supports a hybrid configuration that mixes the setups described in both the
+first and second configurations. How this configuration is set up varies from
+customer to customer. One example of this hybrid configuration lays out orgs
+in a hub and spoke pattern like configuration 2. A spoke on this configuration
+can be composed of single or multiple tenants. See the diagram below for an
+illustrated example of this setup:
+
+<div class="image-at-90">
+
+![Hybrid configuration](/img/multi-tenancy/hybrid-configuration.png
+ "Diagram illustrating the configuration of a hybrid configuration")
+
+</div>
+
+### When to use this option
+
+Depending on the requirements, business customers may require the data
+isolation, customizations, and other features provided by a single org.
+Other customers may not have these demands and can be placed on a shared
+org with other customers. This hybrid configuration supports both types of
+customers.
+
+## Configuration 4
+
+Host tenants in a single org not using Universal Directory
+
+### Summary
+
+In this configuration, the customer assumes responsibility for managing
+users, groups, and application access outside of the Okta org. Okta’s Universal
+Directory (UD) isn't used to store the tenants’ users and groups. See below
+for an illustrated example of this
+configuration.
+
+<div class="image-at-80">
+
+![Hybrid configuration](/img/multi-tenancy/custom-centric-configuration.png
+ "Diagram showing a full custom implementation")
+
+</div>
+
+## Reference
+
+### Additional resources
+
+Additional multi-tenant resources are below:
+
+* [The Secret Features of Okta Access Gateway: Part 1: Multi-data Center and Multi-Tenancy](https://www.okta.com/blog/2020/02/the-secret-features-of-okta-access-gateway-part-1-multi-data-center-and-multi-tenancy-1/)
+
+### Relevant terminology
+
+#### Org
```

Comment:
```suggestion
#### Org
```

Any information that isn't already contained in our Data Model doc should just be added there.

Reply:
Changes will be included in the next version of the multi-tenant documentation.  See https://oktainc.atlassian.net/browse/OKTA-386182 for more info.